microcloud/cmd/microcloud/main_init.go

```diff
@@ -139,11 +139,15 @@ func (c *cmdInit) Run(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
-func lookupPeers(s *service.Handler, autoSetup bool, subnet *net.IPNet, systems map[string]InitSystem) error {
+func lookupPeers(s *service.Handler, autoSetup bool, subnet *net.IPNet, expectedSystems []string, systems map[string]InitSystem) error {
```

Comment:
This function would benefit from a doc block defining what the arguments it accepts are. Especially the new argument, what format is the string slice in and does it do?

Ideally one should be able to use a function by reading its doc block without having to inspect its implementation. It also helps reviewers to check description matches implementation ;)

Reply:
Sure, I'll add a comment. In this case it's just a list of hostnames we expect to find. Basically a subset of the keys from the next argument `systems`. 

That said, should one really be using a function without inspecting its implementation, though?

Reply:
Just to clarify, I'm not trying to say comments are bad or anything, just being pedantic :)

Reply:
Done!